src/auth/auth.controller.ts

```diff
@@ -20,29 +32,51 @@ export class AuthController {
     @Req() req: Request & { user: AuthUser },
     @Res() res: Response,
   ): Promise<void> {
+    this.logger.debug(`googleAuthRedirect`);
     try {
-      const user = req.user;
-      this.logger.debug(`googleAuthRedirect [user: ${JSON.stringify(user)}]`);
-      const userEntity: { isNew: boolean; user: Partial<User> } =
-        await this.authService.findOrCreateUser(user);
-      const accessToken: string = await this.authService.generateToken(userEntity.user);
-      const refreshToken: string = await this.authService.generateRefreshToken(userEntity.user);
-      // res.cookie('access_token', accessToken, {
-      //   httpOnly: true,
-      //   secure: true,
-      //   sameSite: 'none',
-      //   // expires: new Date(Date.now() + 1000 * 60 * 60 * 24 * 7),
-      // });
-      res.cookie('refresh_token', refreshToken, {
-        httpOnly: true,
-        secure: true,
-        sameSite: 'none',
-        expires: new Date(Date.now() + 1000 * 60 * 60 * 24 * 7),
-      });
-      res.redirect(`${process.env.FRONT_URL}/auth/callback/?token=${accessToken}`);
+      const user: AuthUser = req.user;
+      const findUser: Partial<UserEntity> =
+        await this.authService.findOneByEmail(user.email);
+      if (typeof findUser === 'undefined' || findUser === null) {
```

Comment:
요것은 이렇게 표현 하는 것과 다른가요?
```suggestion
      if (!findUser) {
```

Reply:
아 해당부분은 strict null 을 공부하다가 써본것인데, repo에서 find못했을 때는 null나오는것이기 때문에 같은 역활을 합니다. 
그리고 저도 아직 저도 익숙하지 않아 적용을 못했는데 언젠간 본 글에서 
if (!user)와 같은 사용법은 지향을 하라고 하더라고요.
! 연산자를 사용한 조건문의 경우 , 코드 가독성면에서 좋지 않기 때문에 
if (user === null)을 쓰는것을 지양하라고 적혀있었습니다.
다만 이런한 점은 스타일에 따른 문제이기 때문에 편하신대로 작성하셔도 괜찮을거 같습니다. (저도 !  연산자 많이 썼거든요ㅎㅎ..)

Reply:
오호 그렇군요! 새로운 걸 알아갑니다. 감사합니다 :)